src/socket.cr

```diff
@@ -79,7 +79,8 @@ class Socket < IO
     end
   end
 
-  protected def initialize(@fd : Int32, @family, @type, @protocol = Protocol::IP, blocking = false)
+  # Creates a Socket from an existing socket file descriptor.
+  def initialize(@fd : Int32, @family, @type, @protocol = Protocol::IP, blocking = false)
```

Comment:
Shouldn't this also use named arguments like the other `initialize` methods in subtypes?

Reply:
There's no conflict on `Socket`.. named arguments are only really necessary for TCP

Reply:
I see. I meant for consistency, but it's not a big deal.